thirdeye/thirdeye-frontend/app/styles/components/rootcause-anomaly.scss

```diff
@@ -71,7 +71,7 @@
 
   &__stats-wrapper {
     display: flex;
-    margin: 16px 0;
+    margin: 16px 0 0 0;
```

Comment:
nit: this can be written with 3 values: `margin: 16px 0 0`. See under shorthand property: https://www.w3schools.com/css/css_margin.asp

Reply:
fixed